repositories.yaml

```diff
@@ -630,6 +633,7 @@ applications:
     channels:
       - v1_name: burnham
         app_id: burnham
+        deprecated: true
```

Comment:
The top-level app is marked deprecated already. Does this do anything additional?

Reply:
I'm not sure it makes much difference for applications, actually. For libraries it didn't allow a top-level `deprecated` property, so I may just not have looked for one in the apps.

Reply:
I don't think it does then. Mind removing those on apps then?

Reply:
I mean, I was only following _your_ example here 😆: https://github.com/mozilla/probe-scraper/blame/5888157d681bf91c6cf532bd09e2bf148dacbcd8/repositories.yaml#L1212

But joking aside, if it doesn't do anything then there's no reason to add additional lines to an already long file.  I'll remove the ones I added.